### PR TITLE
Readme updates for deployment

### PR DESCRIPTION
**Pull request by will-dyson-nhsd**
## Description
README updates:
- Fix link to 'Setting a password'
- Update deployment information.

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] CHANGELOG entry

### CODE REVIEW

**Review by the-nathan-smith — APPROVED** [review]

**Review by roshaanbajwa — APPROVED** [review]

**Review by sarawilcox — COMMENTED** [review]
In README updates, it just needs a full stop at the end of this sentence: Finally, edit the description of the release to match the content entered into the changelog

**Review by sarawilcox — COMMENTED** [review]
I'd reword as follows - our usual style is not past tense but imperative.

- Enhance deployment instructions and test note in README.
- Fix link in README.

**Comment by will-dyson-nhsd** [comment]
@sarawilcox Thanks for the comments. Changes have been made.
